README.md

```diff
@@ -83,48 +83,51 @@ If you came here to make changes to the generated TypeScript libraries (e.g. `@g
 you are in the right place! Chances are high you don't need to edit any code, just the
 [Nunjucks](https://mozilla.github.io/nunjucks/) templates located in the `templates` folder.
 
-After you edited the files, make the generator available globally:
+This project is written in TypeScript, but we use [Bazel](https://bazel.build/) for managing dependencies
+(for unrelated reasons), which makes the build process a little bit unusual.
+
+Good news is that you don't really need to learn Bazel to make a quick fix. All bazel commands are
+hidden under the hood, so just use regular `npm` commands and just don't be surprised to see a lot of extra
+output.
+
+To compile the code:
 
 ```sh
 # in gapic-generator-typescript folder
-$ npm install  # install dependencies
-$ npm install -g .
-# make sure gapic-generator-typescript launch script in PATH
+$ npm install      # install dependencies
+$ npm run compile  # build project with Bazel
 ```
 
-You'll need `protoc` in your `PATH` as well, take the latest `protoc-*.zip` from
-their [releases page](https://github.com/protocolbuffers/protobuf/releases).
-Make sure it works:
-
+To run the generator:
 ```sh
-$ protoc --version
-libprotoc 3.7.1  # the exact version does not really matter
+bazel run //:gapic-generator-typescript -- --help
 ```
 
 Checkout `googleapis`, which has a lot of protobuf definitions of real Google Cloud APIs:
 
 ```sh
 $ git clone https://github.com/googleapis/googleapis.git
-$ cd googleapis
+$ export GOOGLEAPIS="`pwd`/googleapis"  # save the path.
 ```
 
 Pick some API, how about `translate` `v3`?
 
 ```sh
 $ mkdir -p /tmp/translate-v3-typescript  # where to put the result
-# from googleapis folder:
-$ gapic-generator-typescript -I . \
-  --output_dir /tmp/translate-v3-typescript \
-  --grpc-service-config google/cloud/translate/v3/translate_grpc_service_config.json \
-  `find google/cloud/translate/v3 -name '*.proto'` \
-  google/cloud/common_resources.proto
+# from gapic-generator-typescript folder:
+$ bazel run //:gapic-generator-typescript -- \
```

Comment:
Please add a note that it is not the final state. Gapic-generator should not be called directly by `bazel run`, otherwise bazel looses most of its value (just becomes a way you manage gapic-generator's dependencies). The binary should be provided as an argument to `custom_proto_library` rule, similarly how it is done in Go microgenerator [go_gapic_library](https://github.com/googleapis/gapic-generator-go/blob/master/rules_go_gapic/go_gapic.bzl#L118). 

Then, all the arguments (like `translate_grpc_service_config.json` or protos themselves) become the sources of the `nodejs_gapic_library` rule itself (see [googleapis language go_gapic_library](https://github.com/googleapis/googleapis/blob/master/google/cloud/language/v1/BUILD.bazel#L112) for an example)

Reply:
Yes, this will be updated as soon as it works this way.

Reply:
For now, updated README.